src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+function cloneColumn(section) {
+  Array.from(section.rows).forEach((row) => {
+    const secondCell = row.cells[1];
+    const lastCell = row.cells[row.cells.length - 1];
+
+    if (secondCell && lastCell) {
```

Comment:
This conditional check may not be necessary. If the row doesn't have a second or last cell, it might indicate a problem with your HTML structure. It's generally better to fix such issues directly in the HTML.